doc/general_documentation/INSTALL.md

```diff
@@ -74,16 +74,20 @@ by running 'sudo apt-get update' and then 'sudo apt-get upgrade'*
 ## Configuring the build
 
 ### Note for the build setup
-By default, the configure script uses the following paths:
+
```

Comment:
I tried to add a bit of intro for the user to understand a bit of how a workspace works, is it clear?